server/lib/emailTemplates.js

```diff
@@ -62,6 +62,7 @@ handlebars.registerHelper('moment', (value, props) => {
 
 handlebars.registerHelper('currency', (value, props) => {
   const { currency, precision } = props.hash;
+  if (!currency) return value;
```

Comment:
What's the scenario where we send a value without currency? Just like last time, it's going to send out emails without currency attached.